### DISCUSSION TITLE
Is there a roadmap?

### DISCUSSION THREAD

**Discussion by samnj** [flxzt/rnote#1018] [Development]
I use the app to teach math and I really enjoy it, but it's lacking some key features like different line shapes (dashed) and shapes in general (parabola). It'd be great knowing if any of these are a priority.

I also think it'd be cool to create your own shapes or graphs and save it or "stick it" for later use. That'd be a flexible solution I believe and I could propose it as an enhancement if it makes sense

**Comment by PhilDevProg** [comment]
There's already a merge request for parabola support (#1016) and an issue for an object library (#620)

**Reply by samnj** [reply]
great stuff! Thank you

**Comment by mahdihasan2006** [comment]
What about roadmap? Is there any?
